src/content/self-hosted/guides/azure/index.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Use Azure as your Authentication Provider
+description: Guide on how use Azure as the authentication provider of your Okteto Self-hosted instance
+id: identity-provider-azure
+---
+
+import Image from '@theme/Image';
+import TiersList from '@theme/TiersList';
+
+#  Use Azure as your Identity Provider<TiersList tiers="Self-Hosted" />
+
+This guide will walk you through the steps necessary to configure Azure as the authentication provider for your instace of Okteto.
+
+Please refer to [Azure's official documentation](https://learn.microsoft.com/en-us/power-apps/maker/portals/configure/configure-openid-settings) on Open ID Connect for more information.
+
+## Prerequisites
+
+- An instance of [Okteto](https://okteto.com/docs/self-hosted/).
+- [Helm](https://helm.sh/docs/intro/install/) 3.0+ installed in your local machine.
+- Access to an [Azure account](https://portal.azure.com) with permissions to registrate applications in Azure Active Directory.
+
+## Create an App Registration
+
+Create a new app integration, using the values below:
+
+<p align="center"><Image src="/docs/self-hosted/guides/azure/images/new-app.png" alt="Create the new App Registration" /></p>
+
+Then, fill in the following values on the “Register an application” window:
+
+1. Name: The name of your app (e.g. Okteto)
+2. Accounts in this organizational directory only (Default Directory only - Single tenant)
+3. Redirect URI: https://okteto.{YOUR_DOMAIN}/auth/callback
+
+Once the application has been created, copy the `Application (client) ID` value. This will be your `<<client-id>>`.
+You'll need it to configure your Okteto Instance.
+
+<p align="center"><Image src="/docs/self-hosted/guides/azure/images/client-id.png" alt="Copy the Tenant ID" /></p>
+
+Also, copy the `Directory (tenant) ID` value. This will be your `<<tenant-id>>`.
```

Comment:
same as above. since we haven't told them what the tenant-id is, i don't think we should mention it. and if we do, we should use the same format than in other guides.

```suggestion
Copy the `Directory (tenant) ID` value`. You'll need it to configure your Okteto Instance.
```